docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1268,226 +1268,259 @@ The provided replicas <1>, labels <2> and environment variables <3>  were retain
 
 [NOTE]
 ====
-If the resource name does not match the application name (or the overridden name) instead of reusing the resource a new one will be added. Same goes for the container.
-If the name of the container does not match the application name (or the overridden name), container specific configuration will be ignored.
+* When the resource name does not match the application name (or the overridden name) rather than reusing the resource a new one will be added. Same goes for the container.
+
+* When the name of the container does not match the application name (or the overridden name), container specific configuration will be ignored.
 ====
 
 == Service Binding [[service_binding]]
 
-Quarkus supports binding services to applications via the link:https://github.com/k8s-service-bindings/spec[Service Binding Specification for Kubernetes].
-Specifically Quarkus implements the link:https://github.com/k8s-service-bindings/spec#workload-projection[Workload Projection] part of the specification, thus allowing
-applications running in appropriately configured Kubernetes clusters to consume services (such as a Database or a Broker) without the need for user configuration.
+Quarkus supports the link:https://github.com/k8s-service-bindings/spec[Service Binding Specification for Kubernetes]to bind services to applications.
 
-Currently, the following Quarkus extensions (and extensions that depend on these) support this feature:
+Specifically, with Quarkus you can implement the link:https://github.com/k8s-service-bindings/spec#workload-projection[Workload Projection] part of the specification, thus allowing applications that run in appropriately configured Kubernetes clusters to consume services, such as a Database or a Broker, without the need for user configuration.
```

Comment:
Support for workload projection is implemented by the `kubernetes-service-binding` extension it is not something the user will implement.

Reply:
Sorry, you are right, Will address.